src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchSequenceChecksums.ts

```diff
@@ -33,26 +35,56 @@ export type TranscriptSequenceMetadata = {
   };
 };
 
+export type GeneSequenceMetadata = {
+  stable_id: string;
+  unversioned_stable_id: string;
+  transcripts: TranscriptSequenceMetadata[];
+};
+
+type TranscriptInQuery = {
```

Comment:
The name of the type is misleading here. The query is something that we send to the api; while this type is describing what has been returned from the api. Is it `TranscriptInGeneQueryResult`? Or perhaps `TranscriptInResponse`?

Reply:
✅